Week2/wadeea.js

```diff
@@ -0,0 +1,246 @@
+// 1. Arabic - English - Dutch - Italian -Spanish - French.
+console.log(
+  `مرحبا أيها العالم
+  Hello, World!
+  Hallo, wereld!
+  Ciao, mondo!
+  Hola, mundo!
+  Bonjour, monde`,
+);
+
+// 2. The error was because of using the same quotes as the apostrophe in 'I'm'.
+console.log(`I'm awesome`);
+
+// 3.1
+let x;
+// 3.2
+console.log(`The value of my variable x will be: undefined`);
+// 3.3
+console.log(x);
+// 3.4 If I initialize the variable x here, I will get a syntax error,
+// because It's already been declared. So I've only defined it.
+x = 7;
+// 3.5
+console.log(`The value of x will be: 7`);
+// 3.6
+console.log(x);
+
+// 4.
+let y = `This is a string`;
+// 4.1
+console.log(`The value of the string will be the following sentence: This is a string`);
+// 4.2
+console.log(y);
+// 4.3
+y = `New string`;
+// 4.4
+console.log(`The value will be the following: New string`);
+// 4.5
+console.log(y);
+
+// 5. The nearest integer is 7 because it's less than 7.50
+// 5.1
+let z = 7.25;
+// 5.2
+console.log(z);
+// 5.3
+let a = 7;
+// 5.4
+console.log(a);
+// 5.5
+let theHighestValue;
+if (a > z) {
+  theHighestValue = a;
+} else {
+  theHighestValue = z;
+}
+// 5.6
+console.log(`The highest value is: ` + theHighestValue);
+
+// 6.1
+const citiesInNetherlands = [];
+// 6.2
+console.log(`The value of the array is: 0`);
+// 6.3
+console.log(citiesInNetherlands);
+// 6.4
+const favoriteAnimals = [`cat`, `dog`, `hamster`, `lion`, `tiger`];
+// 6.5
+console.log(favoriteAnimals);
+// 6.6
+favoriteAnimals.push(`baby pig`);
+// 6.7
+console.log(favoriteAnimals);
+
+// 7.1
+const myString = `this is a test`;
+console.log(myString);
+// 7.2
+myString.length;
+// 7.3
+console.log(myString.length);
+
+// 8.1
+const myFirstName = `Wadeea`;
+const myLastName = `Kiwan`;
+let myAge = 33;
```

Comment:
Why did you decide to go with `let` for `age`?

Reply:
It was maybe from the last week that we wanted to manipulate it, but it's not the case here. I'll change it to const.